docs/tbdev_getting_started.ipynb

```diff
@@ -12,7 +12,7 @@
     },
```

Comment:
<p>Uploading the TensorBoard logs will give<strong> you a URL </strong>that can be shared with anyone.</p><p> </p><p>Uploaded TensorBoards are public, so do not upload sensitive data.</p><p> </p><p>&lt;Also ...&gt;</p><p> </p><p>This sentence is a run on:</p><p>    The uploader will exit when it has completed uploading the entire <code>logdir</code>, this is the function of the <code>--one_shot</code> flag. </p><p> </p><p>Rephrase to:</p><p> </p><p>The uploader will exit when the entire logdir has been uploaded. (This is what the --one_shot flag specifies.)</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/tensorflow/tensorboard/pull/3920/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='10'/>